validator/accounts/wallet_create.go

```diff
@@ -278,3 +287,25 @@ func inputKeymanagerKind(cliCtx *cli.Context) (keymanager.Kind, error) {
 	}
 	return keymanager.Kind(selection), nil
 }
+
+func inputNumAccounts(cliCtx *cli.Context) (int64, error) {
```

Comment:
should this function still be here? if it's moved to wallet_recover?

Reply:
still needed in both packages, see above comment!